Automattic/wp-calypso
# Customize Jetpack connection logged-in screen for woocommerce core profiler flow

Related to https://github.com/woocommerce/team-ghidorah/issues/213

## Proposed Changes

* Customize jetpack connection (Logged-in) screen for woocommerce core profiler flow

![Screenshot 2023-06-07 at 14 55 34](https://github.com/Automattic/wp-calypso/assets/4344253/c483a9e1-a606-47bf-bc2b-df57870f8b80)



## Testing Instructions



* Create a JN site
* Go to OBW and continue until Jetpack connection step (/jetpack/connect/authorize)
* Replace the host `wordpress.com` with your Calypso host
* Change query parameter `from` to `woocommerce-core-profiler` and add `installed_ext_success=1` query parameter
* Observe the `Extensions successfully installed!` notice is shown and automatically dismissed
* Confirm the screen matches the design (Y5pUYSJPsGEud1vknUZhi8-fi-739_54533)
* Reload the page
* Observe the `Extensions successfully installed!` is not shown.
* Enable track debugging `localStorage.setItem( 'debug', 'calypso:analytics' );`

**Scenario: User clicks on Sign in as a different user**

1. Open dev tools console and tick `Preserve log` option
2. Clicks on `Sign in as a different user`
3. You should be redirected to login page.
4. Observe that `calypso_jpc_different_user_click` event is recorded

**Scenario: User clicks on No, Thanks**

1. Go back to Jetpack connection page
2. Clicks on `No, Thanks` button
3. You should be redirected to `WooCommerce -> Home`
4.  Observe that `calypso_wc_coreprofiler_jpc_skip` event is recorded

**Scenario: User clicks on Connect your account button**

1. Go back to Jetpack connection page
2. Clicks on `Connect your account` button
5. Connect  to Jetpack
6. You should be redirected to `WooCommerce -> Home`
7. Observe that `calypso_jpc_approve_click` event is recorded




## Pre-merge Checklist
